server/integration_tests/standalone/adversarial.py

```diff
@@ -311,9 +317,13 @@ def run_queries_from_file(self, input_csv_file, output_csv_file):
           csv_writer.writerow(result.to_csv_row())
 
   def run_query(self, query: str) -> Result:
+    sdg_param = ""
+    if self.is_sdg:
+      sdg_param = "sdg"
     result = unknown_result(
         query, self.base_url +
-        f'/explore#q={urllib.parse.quote_plus(query)}&llm_api={self.llm_api}')
+        f'/explore#q={urllib.parse.quote_plus(query)}&llm_api={self.llm_api}&dc={sdg_param}'
```

Comment:
For main dc, this will make a request with `&dc=`. Would it be cleaner to not encode the `dc` parameter at all in that case?

Reply:
Done